skatepark/theme.json

```diff
@@ -1,6 +1,12 @@
 {
 	"$schema": "https://json.schemastore.org/theme-v1.json",
 	"version": 1,
+	"templateParts": [
+		{
+			"name": "header-ollie",
+			"area": "header"
```

Comment:
this one is not on Blockbase? Since Skatepark has not been yet launched, can we simplify the header so it uses one of the others that we've already defined? Mainly to avoid the extra CSS to maintain if at all possible.

/cc @kjellr

Reply:
Or we could move it to Blockbase to give all themes another option. It would be nice to be able to achieve it with less CSS though.

Reply:
I would really like to add it as another option to Blockbase too.  

At the same time I was also keen to evaluate/demonstrate what the template part option added in a child theme would behave.  I don't think it's a bad experience and works for this pass (I hope).  It just felt too daunting compared to the other headers to do that now though.  I definitely don't want to trade that header in for one of the others; the design is solid and it's a really good option I think.

Reply:
We can move it to Blockbase on a follow up PR. I still think it needs some work, there's few couple of issues opened against it because it's not looking like the design: https://github.com/Automattic/themes/issues/4965 https://github.com/Automattic/themes/issues/4591 https://github.com/Automattic/themes/issues/4964